docs/interactions/structured-messages/rich-link.md

```diff
@@ -34,7 +34,7 @@ curl -X POST "https://[YOUR DOMAIN].api.digital.ringcentral.com/1.0/contents"
 | **`structured_content`** | Object | Payload of the structured message. |
 | **Structured Content Settings** | | |
 | **`structured_content.type`** | String | Type of the structured message. Must be "rich_link". |
-| **`structured_content.url`** | String | Url of the rich link. Limited to 2048 characters. Should only have http and https schemes. If the device does not support structured messages, this field will be sent as the message. |
+| **`structured_content.url`** | String | Url of the rich link. Limited to 2048 characters. Should only have http and https schemes except for Apple Messages for Business were it could starts with '?' ([more informations](#example-apple-messages-for-business) ). If the device does not support structured messages, this field will be sent as the message. |
```

Comment:
```suggestion
| **`structured_content.url`** | String | Url of the rich link. Limited to 2048 characters. Should only have http and https schemes except for Apple Messages for Business were it could start with '?' ([more informations](#example-apple-messages-for-business) ). If the device does not support structured messages, this field will be sent as the message. |
```

Reply:
Actually you should just add a new line in the "Properties Unique to this Channel" table of AMB section. This is where we're putting the differences with the generic fields